packages/kit-bg/src/vaults/impls/btc/Vault.ts

```
@@ -128,8 +128,40 @@ export default class VaultBtc extends VaultBase {
+        const accountAddress = await this.getAccountAddress();
+        const { send } = stakingInfo;
+        const action = await this.buildInternalStakingAction({
+          accountAddress,
+          stakingInfo,
+          stakingToAddress: decodedPsbtAction.assetTransfer?.to,
+        });
+
+        let sendNativeTokenAmountBN = new BigNumber(0);
+        let sendNativeTokenAmountValueBN = new BigNumber(0);
+
+        if (send && send.token.isNative) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)